### ISSUE TITLE
Could not strap packages with abnormal  exit code 1

### ISSUE THREAD

**Issue by MacCygwin** [archlinux/archinstall#2318]
![VirtualBox_arch-kde_01_01_2024_19_22_50](https://github.com/archlinux/archinstall/assets/94826164/ff65c485-9bea-4bde-9fd1-56d7dcefca6f)

I used the archinstall script to install and i was connected to internet. I get this error message despite multiple tries. Pls help

**Comment by NikolaM-Dev** [comment]
Same problem here, I have tried several times, installing the most recent ISO, reset the disk from windows, install `ardhlinux-keyring` because I saw it in another post and nothing.

I don't know if there is a way to use a previous version of the installer or something. I am a novice user in terms of operating system and I need to work, I directly do the manual installation or is there a way to use the script?

Thank you very much in advance
